devices.js

```diff
@@ -2273,6 +2273,17 @@ const devices = [
             };
         },
     },
+
+    // Livolo
+    {
+        zigbeeModel: ['TI0001          '],
+        model: 'TI0001          ',
+        description: 'Zigbee switch',
+        vendor: 'Livolo',
+        supports: 'brightness, on/off',
+        fromZigbee: [fz.ignore_onoff_report, fz.livolo_switch_dev_change],
+        toZigbee: [tz.light_brightness, tz.livolo_switch_on_off],
```

Comment:
How does the light brightness work? Can't this only control 1 endpoint while this device has 2?

Reply:
Good, can you remove the brightness converters/supports?

Reply:
I'll do it.